### DISCUSSION TITLE
Episode 161: 🎄 EOP, Rust, Hylo, C++20/23/26/29 & More with Zach Laine (Holiday Special #4)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#53] [Episode]
Episode Link: https://adspthepodcast.com/2023/12/22/Episode-161.html

Feel free to write comments or ask questions 🙂

**Comment by paulfloyd** [comment]
If you like EoP then take look at FM2GP https://www.fm2gp.com/

Stepanov is to C++ libraries what Stevens is to TCP/IP and Unix.
